delivery/meteorsRouter.ts

```diff
@@ -0,0 +1,50 @@
+import express, { Request, Response, NextFunction } from 'express'
+import fetchMeteors from '../useCases/fetchMeteors.ts'
+import getRoverImage from '../useCases/getRoverImage.ts'
+import { getMeteorSchema } from '../validation/schemas/getMeteorSchema.ts'
+import { validateRequest } from '../validation/validator.ts'
+import { postRoverImageSchema } from '../validation/schemas/postRoverImageSchema.ts'
+import { MeteorQuery } from '../types/meteorRequest.ts'
+
+const meteorRouter = express.Router()
+
+meteorRouter.get(
+  '/meteors',
+  validateRequest(getMeteorSchema, 'query'),
+  async (request: Request, response: Response, next: NextFunction) => {
+    try {
+      const meteorQuery: MeteorQuery = request.query
+      const meteors = await fetchMeteors(meteorQuery.date!, meteorQuery.count!, meteorQuery.wereDangerousMeteors!)
```

Comment:
every variable that is not a string must be converted to the correct type before passing it as an argument to the service